apps/trading-e2e/src/integration/market-liquidity.cy.ts

```diff
@@ -0,0 +1,330 @@
+    cy.getByTestId(marketSummaryBlock).within(() => {
+      cy.getByTestId('target-stake').within(() => {
+        cy.getByTestId(itemHeader).should('have.text', 'Target stake');
+        cy.getByTestId(itemValue).should('have.text', '10.00 tDAI').realHover();
+      });
+    });
+    cy.getByTestId('tooltip-content').should(
+      'contain.text',
+      `The market's liquidity requirement which is derived from the maximum open interest observed over a rolling time window.The market's liquidity requirement which is derived from the maximum open interest observed over a rolling time window.`
+    );
+  });
+
+  it('can see supplied stake', () => {
+    // 5002-LIQP-007
+    cy.getByTestId(marketSummaryBlock).within(() => {
+      cy.getByTestId('supplied-stake').within(() => {
+        cy.getByTestId(itemHeader).should('have.text', 'Supplied stake');
+        cy.getByTestId(itemValue).should('have.text', '0.01 tDAI').realHover();
+      });
+    });
+    cy.getByTestId('tooltip-content').should(
+      'contain.text',
+      'The current amount of liquidity supplied for this market.'
+    );
+  });
+
+  it('can see liquidity supplied', () => {
+    //// 5002-LIQP-008
+    cy.getByTestId(marketSummaryBlock).within(() => {
+      cy.getByTestId('liquidity-supplied').within(() => {
+        cy.getByTestId(itemHeader).should('have.text', 'Liquidity supplied');
+        cy.getByTestId('indicator').should('be.visible');
+        cy.getByTestId(itemValue).should('have.text', '0.10%').realHover();
+      });
+    });
+  });
+
+  it('can see market id', () => {
+    // 5002-LIQP-009
+    cy.getByTestId(marketSummaryBlock).within(() => {
+      cy.getByTestId('liquidity-market-id').within(() => {
+        cy.getByTestId(itemHeader).should('have.text', 'Market ID');
+        cy.getByTestId(itemValue).should('have.text', 'market-0');
+      });
+    });
+  });
+
+  it('can see market id', () => {
+    // 5002-LIQP-010
+    cy.getByTestId(marketSummaryBlock).within(() => {
+      cy.getByTestId('liquidity-learn-more').within(() => {
+        cy.getByTestId(itemHeader).should('have.text', 'Learn more');
+        cy.getByTestId(itemValue).should('have.text', 'Providing liquidity');
+        cy.getByTestId('external-link')
+          .should('have.attr', 'href')
+          .and(
+            'include',
+            'https://docs.vega.xyz/testnet/concepts/liquidity/provision'
+          );
+      });
+    });
+  });
+
+  describe('liquidity table view', { tags: '@smoke' }, () => {
+    it('can see table headers', () => {
+      cy.getByTestId('tab-active').within(($headers) => {
+        cy.wrap($headers)
+          .get('.ag-header-cell-text')
+          .each(($header, i) => {
+            cy.wrap($header).should('have.text', headers[i]);
+          });
+      });
+    });
+
+    it('renders liquidity active table correctly', () => {
+      // 5002-LIQP-011
+      cy.get(rowSelectorLiquidityActive)
+        .first()
+        .find('[col-id="party.id"]')
```

Comment:
(non-blocking): some of these selectors may be moved to consts, as they are quite repetitive

Reply:
Done